hypothesis-python/src/hypothesis/internal/compat.py

```diff
@@ -551,14 +551,14 @@ def implements_iterator(it):
 # custom __floor__ or __ceil__ methods may convert via floats.
 # See issue #1667, Numpy issue 9068.
 def floor(x):
-    y = _long_integer_type(x)
```

Comment:
I think `_long_integer_type` is now unused again, so its definitions above can probably be deleted.